Dockerfile.melodic

```diff
@@ -41,9 +41,6 @@ COPY cartographer_rviz/package.xml catkin_ws/src/cartographer_ros/cartographer_r
 COPY scripts/install_debs.sh cartographer_ros/scripts/
 RUN cartographer_ros/scripts/install_debs.sh && rm -rf /var/lib/apt/lists/*
 
-# Install proto3.
-RUN /catkin_ws/src/cartographer/scripts/install_proto3.sh
-
```

Comment:
Remove also  `RUN cartographer_ros/scripts/install.sh --pkg ceres-solver` below and put it into the base build file?